### PR TITLE
Add chain info to the feedback email body

### PR DESCRIPTION
**Pull request by elgatovital**
Handles #1622

@gnosis/mobile-devs

### CODE REVIEW

File: app/src/main/java/io/gnosis/safe/ui/settings/app/GetInTouchFragment.kt [hunk 1]
```diff
@@ -116,6 +116,7 @@ class GetInTouchViewModel
         val text = StringBuilder()
         text.appendln("${context.getString(R.string.app_name)} v${BuildConfig.VERSION_NAME}")
         text.appendln(context.getString(R.string.feedback_safe, activeSafe?.address?.asEthereumAddressChecksumString() ?: ""))
+        text.appendln(context.getString(R.string.feedback_chain, activeSafe?.chain?.name ?: ""))
```
Comment (biafra23) [inline app/src/main/java/io/gnosis/safe/ui/settings/app/GetInTouchFragment.kt:119]: Is it possible to have more than one chain with the same name? If so, then we should include the chain id.

**Review by biafra23 — APPROVED** [review]
Maybe the chain id would be useful too?

File: app/src/main/java/io/gnosis/safe/ui/settings/app/GetInTouchFragment.kt (hunk 1 above)
Comment (elgatovital) [reply app/src/main/java/io/gnosis/safe/ui/settings/app/GetInTouchFragment.kt:119]: Chain: <chain name> (<id>) ?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -235,6 +235,7 @@
     <string name="feedback_subject">Feedback</string>
 
     <string name="feedback_safe">Safe address: %1$s</string>
+    <string name="feedback_chain">Chain: %1$s (%2$s)</string>
```
Comment (sche) [inline app/src/main/res/values/strings.xml:238]: Will it be "Chain:  (nil)" when no safe is selected @elgatovital ?

Comment (biafra23) [reply app/src/main/res/values/strings.xml:238]: No it is `Chain: (null)` :-)

**Pull request merged**